.circleci/config.yml

```diff
@@ -0,0 +1,48 @@
+version: 2.1
+
+jobs:
+  build:
+    docker:
+      - image: circleci/ruby:2.6.5-node-browsers
+      - image: circleci/postgres:13.2
+        environment:
+          POSTGRES_USER: 'postgres'
+          POSTGRES_HOST_AUTH_METHOD: trust
+
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+          - v1-dependencies-{{ checksum "Gemfile.lock" }}
+
+      - run:
+          name: install dependencies
+          command: |
+            gem install bundler
+            bundle install --clean --jobs=4 --retry=3 --path vendor/bundle
+
+      - save_cache:
+          paths:
+            - vendor/bundle
+          key: v1-bundle-dependencies-{{ checksum "Gemfile.lock" }}
+
+      # Database setup
+      - run: cp .circleci/database.yml config/database.yml
+      - run: RAILS_ENV=test bundle exec rake db:create
+      - run: RAILS_ENV=test bundle exec rake db:migrate
+
+      # Run tests
+      - run:
+          name: run tests
+          command: RAILS_ENV=test bundle exec rspec
+      
+      - store_test_results:
+          path: /tmp/test-results
+      - store_artifacts:
+          path: /tmp/test-results
+          destination: test-results
+
+      #  Run annotate if missing
```

Comment:
I am not sure of this.
Probably you would run `migrate` command in your local machine, generating all annotations needed.

Reply:
Yeah, I was thinking that, but I added it just in case. I'll remove it :)